examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -21,8 +21,8 @@
 
 Note that for an actual predictive modeling study of aging, the study
 should be ran on the full set of subjects. Also, parameters such as the
-smoothing applied to the data and the number of features selected by the
-Anova step should be set by nested cross-validation, as they impact
```

Comment:
Rephrase as:

Also, all parameters should be set by cross-validation. This includes the smoothing applied to the data and the number of features selected by the Anova step. Indeed, even these data-preparation parameter impact significantly the prediction score.